integration_tests/integration_tests/tests/dbt_project.py

```diff
@@ -180,9 +180,10 @@ def seed(self, data: List[dict], table_name: str):
     @contextmanager
     def create_temp_model_for_existing_table(self, table_name: str):
         model_path = TMP_MODELS_DIR_PATH.joinpath(f"{table_name}.sql")
-        model_path.touch()  # Just need the file to exist, the contents doesn't matter
+        model_path.write_text("SELECT 1 AS col")
```

Comment:
I needed to run the model to generate metrics for it.
You cannot run an empty model.